<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTableDialog</class>
 <widget class="QDialog" name="AddTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un tableau</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_display_tab">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="m_adjust_table_size_cb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Ajuster la taille du tableau au folio</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_add_table_and_folio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Ajouter de nouveau folio et tableau si nécessaire.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_table_name_le">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Nom du tableau</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Texte des en-têtes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QComboBox" name="m_header_alignment_cb">
            <item>
             <property name="text">
              <string>Gauche</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Centre</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Droite</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Police :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="m_edit_header_margins_pb">
            <property name="text">
             <string>Éditer</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="m_header_font_pb">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Marges :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Alignement :</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Texte du tableau</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Marges :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="m_table_margins_pb">
            <property name="text">
             <string>Éditer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Alignement :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="m_table_alignment_cb">
            <item>
             <property name="text">
              <string>Gauche</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Centre</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Droite</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Police :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="m_table_font_pb">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ConfigSaveLoaderWidget" name="m_config_gb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigSaveLoaderWidget</class>
   <extends>QGroupBox</extends>
   <header>configsaveloaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_adjust_table_size_cb</tabstop>
  <tabstop>m_add_table_and_folio</tabstop>
  <tabstop>m_table_name_le</tabstop>
  <tabstop>m_edit_header_margins_pb</tabstop>
  <tabstop>m_header_alignment_cb</tabstop>
  <tabstop>m_header_font_pb</tabstop>
  <tabstop>m_table_margins_pb</tabstop>
  <tabstop>m_table_alignment_cb</tabstop>
  <tabstop>m_table_font_pb</tabstop>
  <tabstop>m_tab</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_button_box</sender>
   <signal>accepted()</signal>
   <receiver>AddTableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_button_box</sender>
   <signal>rejected()</signal>
   <receiver>AddTableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
